package.json

```diff
@@ -5,15 +5,10 @@
     "email": "msopentech@microsoft.com",
     "url": "https://github.com/AzureAD/azure-activedirectory-library-for-js"
```

Comment:
This should be switched to Microsoft. how about nugetaad@microsoft.com? Thats what we add in our wilson nuget packages. Now that we are using this email address for npm, makes sense to use that email address here too.